### PR TITLE
Make //verify:verify-pylint pass on a clean bazel container

### PR DESCRIPTION
**Pull request by fejta**
ref #3037

Also fix lint errors in various files and add verify/*.py to lint list
/assign @spxtr

### CODE REVIEW

File: verify/pylint_bin.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# limitations under the License.
+
+import os
+import sys
+
+import pylint
+
+
+if __name__ == '__main__':
+    # Otherwise bazel's symlinks confuse pylint/astroid
```
Comment (fejta) [inline verify/pylint_bin.py]: OMG was this every painful to debug

Comment (spxtr) [inline verify/pylint_bin.py]: How did you figure this out?

**Comment by spxtr** [comment]
Nice.
/lgtm

File: verify/verify-pylint.sh [hunk 2]
```diff
@@ -18,14 +18,15 @@ set -o nounset
...
-pylint scenarios/*.py
-pylint jenkins/*.py
-pylint jobs/*.py
-pylint metrics/*.py
-pylint queue-health/graph/graph.py
-pylint queue-health/weekly_commit_stats.py
-pylint triage/*.py
-pylint experiment/*.py
+$pylint \
+  scenarios/*.py \
```
Comment (ixdy) [inline verify/verify-pylint.sh]: do we still need these lists? can we just `find` instead?

Comment (spxtr) [inline verify/verify-pylint.sh]: It won't pass on certain files still, notably `gubernator/*.py`.

Comment (ixdy) [inline verify/verify-pylint.sh]: ah, good reason. we could turn this into an exclude list, though.

Comment (krzyzacy) [inline verify/verify-pylint.sh]: I have https://github.com/kubernetes/test-infra/pull/2950

File: verify/pylint_bin.py (hunk 1 above)
Comment (fejta) [inline verify/pylint_bin.py]: 70% grit, 30% `import pdb; pdb.set_trace(); print 'erick'`

**Pull request merged**